src/styles/header.scss

```diff
@@ -0,0 +1,62 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  background: $background-color;
+  box-shadow: 0 2px 4px 0 #0000000d;
+}
+
+.logo {
+  display: block;
+
+  height: 40px;
+  width: 40px;
+}
+
+.nav {
```

Comment:
actually, it's a separate block, so according to BEM rules, it should be in a separate file. Same for `.logo` and `.range` (but you could do them elements of `header` and `product-cards` blocks accordingly)